tox.ini

```diff
@@ -35,4 +35,4 @@ basepython = python3.9
 extras = 
     test
 commands =
-    python -m pytest skrf/tests/test_network.py skrf/tests/test_vectorfitting.py 
\ No newline at end of file
+    python -m pytest {posargs}
```

Comment:
Following your PR, all the setup is no more required, ins't it?

Reply:
I think it's required to test, if our function will work without matplotlib installed, but we don't have to specify a subset of the tests. Testcases which require matplotlib are skipped in this case.